### DISCUSSION TITLE
Switch between different globals.local

### DISCUSSION THREAD

**Discussion by nya1** [netblue30/firejail#5588] [Q&A] [answered]
Hello,
I have different `globals.local` based on the environment that I want to run, is it possible to switch between them via some CLI option?
I tried to use `--private=/home/username/workspace/testing` and I have the globals.local file placed in the private home .config (`/home/username/workspace/testing/.config/firejail/globals.local`) but it seems this is ignored by firejail

**Comment by rusty-snake** [comment, marked as answer]
1. `--private` does not modify the behaviour of firejail, the same way `--chroot` does not read from a different `/etc`.
2. What do you mean with "based on the environment that I want to run"? How do you use them? What's in them? Asking to avoid XY Problems.
3. You can include additional files with `--include=~/.config/firejail/testing.local`.

**Reply by nya1** [reply]
Switching the `globals.local` with the `--include` did the trick! thanks a lot
